.vib/spring-cloud-dataflow-composed-task-runner/goss/spring-cloud-dataflow-composed-task-runner.yaml

```diff
@@ -0,0 +1,14 @@
+command:
+  check-scdf-composed-task-runner-version:
+    timeout: 20000
+    exec: timeout --preserve-status 15 /opt/bitnami/java/bin/java -jar /opt/bitnami/spring-cloud-dataflow-composed-task-runner/scdf-composed-task-runner.jar
```

Comment:
Same feedback as in https://github.com/bitnami/containers/pull/34286/files#r1200343976